### PR TITLE
Add users for collectd

### PR DESCRIPTION
**Pull request by mvdriel**

### CODE REVIEW

File: tasks/collectd.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# tasks file for influxdb
+---
+- name: collectd | create collectd_auth_file
+  template:
+    src: etc/influxdb/collectd_auth_file.j2
+    dest: /etc/influxdb/collectd_auth_file
+    owner: "{{ item.owner | default('root') }}"
+    group: "{{ item.group | default('root') }}"
+    mode: "{{ item.mode | default('0644') }}"
```
Comment (tersmitten) [inline tasks/collectd.yml]: `mode` should probably be changed

Comment (tersmitten) [reply tasks/collectd.yml]: or maybe `owner` and / or `group`

File: tasks/collectd.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  when: influxdb_collectd_users | default([]) | length > 0
+  notify: restart influxdb
+  tags:
+    - influxdb-collectd-create
+    - influxdb-collectd-create-auth-file
+
+- name: collectd | remove collectd_auth_file
+  file:
+    path: /etc/influxdb/collectd_auth_file
```
Comment (tersmitten) [inline tasks/collectd.yml]: Can you put this in a variable like this?

https://github.com/Oefenweb/ansible-apt/blob/master/vars/main.yml#L3

Comment (tersmitten) [reply tasks/collectd.yml]: And then use it like this?

https://github.com/Oefenweb/ansible-apt/blob/master/tasks/main.yml#L19

File: tasks/collectd.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# tasks file for influxdb
+---
+- name: collectd | create collectd_auth_file
+  template:
+    src: "{{ influxdb_collectd_auth_file | ltrim('/') }}".j2
+    dest: "{{ influxdb_collectd_auth_file }}"
+    owner: influxdb
```
Comment (tersmitten) [inline tasks/collectd.yml]: `{{ influxdb_user }}`

File: tasks/collectd.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    group: influxdb
```
Comment (tersmitten) [inline tasks/collectd.yml]: `{{ influxdb_group }}`

**Comment by tersmitten** [comment]
Nice! I'm interested in how you use it. This way we can send collectd data securely in an insecure network, right?

**Comment by mvdriel** [comment]
@tersmitten That's right! You can use the added example. Let me know if you have any questions.

**Comment by tersmitten** [comment]
I understand the example, but there's also "the other side", right?

**Comment by mvdriel** [comment]
@tersmitten Yes, that's right. It works perfectly together with your other role. Configuration is exactly this example: https://github.com/Oefenweb/ansible-collectd#with-encryption

**Comment by tersmitten** [comment]
image

**Pull request merged**